api/v1/runtime_types.go

```diff
@@ -197,10 +197,15 @@ type Filter struct {
 	Egress  Egress   `json:"egress"`
 }
 
+// Ingress filtering can be enabled for `shoot-networking-fitler` extension with
+// the blackholing feature, see https://github.com/gardener/gardener-extension-shoot-networking-filter/blob/master/docs/usage/shoot-networking-filter.md#ingress-filtering
 type Ingress struct {
+	// It means that the blackholing filtering is enabled.
+	// Either on the per shoot level, or the per worker group level.
```

Comment:
Pls remove that line

Reply:
done in https://github.com/kyma-project/infrastructure-manager/pull/712/commits/1e5ef12a720f85f0bb735bb63427cc8ff0180ed6